### ISSUE TITLE
Optional "Pause+Show interface / Resume+Hide interface" touch action

### ISSUE THREAD

**Issue by SameenAhnaf** [TeamNewPipe/NewPipe#5955]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Video interface needs to popup before pausing a video. Although pausing video is a common action, it cannot be done within a single tap. Adding an option to enable "Pause+Show interface / Resume+Hide interface" could easily solve the problem.

#### How will you/everyone benefit from this feature?

Users will be able to pause videos within a single tap. Pausing the video is a common action. So, it will remove a lot of hassle.

**Comment by mhmdanas** [comment]
Although this doesn't exactly fix this issue, you can double-tap in the middle of the player to pause/resume the video without having to wait for the pause/resume button to be shown.

**Comment by SameenAhnaf** [comment]
@mhmdanas Thanks for the suggestion. However, pausing or resuming is not the activity I need. I need other activities on the interface too. This would have been appreciated if users could choose touch action.

**Comment by mhmdanas** [comment]
@SameenAhnaf hence why I said "Although this doesn't exactly fix this issue,"

**Comment by SameenAhnaf** [comment]
@mhmdanas I wanted to thank you. That's why, I at mentioned you. Never mind if you feel disturbed.

**Comment by mhmdanas** [comment]
No, I wasn't disturbed or anything; I was just trying to clarify my original comment.
